src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -62,6 +62,44 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
         } private set;
     }
 
+    /*********************************************************************************************************
+    * @description Override saveSettings from STG_Panel to update Custom Setting Error Setting
+    * @return PageReference 
+    */
+    public override PageReference saveSettings() {
+        STG_SettingsManager_CTRL.idPanelCurrent = idPanel();
+        Savepoint sp = Database.setSavepoint();
+        try {
+            List<SObject> toUpdate = new List<SObject>();
+
+            if (STG_Panel.stgService.stgErr != null && !Test.isRunningTest()) {
+                toUpdate.add(STG_Panel.stgService.stgErr);
+            }
+
+            Boolean hasUpdatePermission = UTIL_Describe.getObjectDescribe(
+                String.valueOf(Error_Settings__c.SObjectType)
+            ).IsUpdateable();
+
+            if (!hasUpdatePermission) {
+                throw new DmlException(
+                    String.format(
+                        System.Label.flsError,
+                        new List<String>{System.Label.stgLabelErrorSettings})
+                );
+            }
+
+            update toUpdate;
+            listDR = new List<DetectResult>();
```

Comment:
Let's add a comment notifying that health check results will be cleared out when settings change.